clients/cpp-console/src/common/AgentConfiguration.cpp

```diff
@@ -63,21 +63,14 @@ shared_ptr<AgentConfiguration> AgentConfiguration::LoadFromFile(const string& pa
     config->_logFilePath = j.value(FieldNames::LogFilePath, "");
     config->_volume = atoi(j.value(FieldNames::Volume, "").c_str());
 
-	if (config->_keywordModelPath.length() > 0)
-	{
-		if (!std::experimental::filesystem::exists(config->_keywordModelPath))
-		{
-			config->_loadResult = AgentConfigurationLoadResult::KWFileNotFound;
-			return config;
-		}
-
-		std::experimental::filesystem::path pathObj(config->_keywordModelPath);
-		if (!pathObj.has_extension() || pathObj.extension().string() != ".table")
-		{
-			config->_loadResult = AgentConfigurationLoadResult::KWFileWrongExtension;
-			return config;
-		}
-	}
+    if (config->_keywordModelPath.length() > 0)
+    {
+        if (!std::experimental::filesystem::exists(config->_keywordModelPath))
+        {
+            config->_loadResult = AgentConfigurationLoadResult::KWFileNotFound;
+            return config;
+        }
+    }
 
```

Comment:
Regarding table file extension: If we had APIs to validate KWS table file (make sure it's valid, get version, get keyword, etc.) I would say forget about the file extension. If we used a wrong table file, and Speech SDK returned an error when we load the "table" file (instead of silentley failing), I would say forget about the file extension. But since we don't have either, maybe keeping the .table file check is a good sanity check to have, and it will be documented that you need to copy over one of the .table files (or create your own via the KWS portal).
My suggestion is to keep this check for .table extension.

Reply:
removed check